Gems/Atom/Feature/Common/Assets/Passes/SMAA1xApplyLinearHDRColor.pass

```diff
@@ -25,10 +25,7 @@
                 {
                     "Name": "OutputColor",
                     "SlotType": "Output",
-                    "ScopeAttachmentUsage": "RenderTarget",
-                    "LoadStoreAction": {
-                        "LoadAction": "DontCare"
-                    }     
```

Comment:
why remove the load action? Just want to validate as LoadAction Don't Care can be an optimization on mobile gpus.